docs/concepts/storage/storage-classes.md

```diff
@@ -52,6 +52,21 @@ mountOptions:
   - debug
 ```
 
+### Volume Binding Mode
+{% assign for_k8s_version="v1.9" %}{% include feature-state-alpha.md %}
+
+This field requires the `VolumeScheduling` feature gate to be enabled.
+
+A new field, `volumeBindingMode`, controls if volume binding should happen
+immediately or wait until pod scheduling.  Valid values are `Immediate` and
+`WaitForFirstConsumer`.  The default mode is `Immediate`, which is the same
+behavior prior to this feature.
+
+**Note:** The `WaitForFirstConsumer` volume binding mode does not support
+dynamic provisioning yet.  Also, it currently is only useful for the
+[local](volumes.md#local) volume type.  Additional volume types will be
```

Comment:
ah ... I see. This is the only use case. Makes sense now .

I'd suggest we avoid document "predictions" about future, so the last sentence can be removed?

Reply:
Since this feature is currently only useful for local volumes right now, I could move all of this under the local volume section instead.  And when we support more volume types in the future, then I can move it out.